Description: fix mocha path
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-11-14

--- a/test.ts
+++ b/test.ts
@@ -15,7 +15,7 @@
   afterEach(() => Error.stackTraceLimit = originalStackTraceLimit);
 
   it('gets current caller file', function() {
-    expect(ensurePosix(getCallerFile())).to.eql(ensurePosix(__dirname + '/node_modules/mocha/lib/runnable.js'));
+    expect(ensurePosix(getCallerFile())).to.eql(ensurePosix('/usr/share/nodejs/mocha/lib/runnable.js'));
   });
 
   it('gets current file, as it is the caller', function() {
